lightning/src/chain/keysinterface.rs

```diff
@@ -406,8 +407,28 @@ pub trait KeysInterface {
 
 	/// Get secret key material as bytes for use in encrypting and decrypting inbound payment data.
 	///
+	/// If the implementor of this trait supports [phantom node payments], then every node that is
+	/// intended to be included in the phantom invoice(s) route hints must return the same value from
+	/// this method.
+	///
 	/// This method must return the same value each time it is called.
+	///
+	///[phantom node payments]: crate::ln::channelmanager::ChannelManager::get_phantom_scid
 	fn get_inbound_payment_key_material(&self) -> KeyMaterial;
+
+	/// Get the phantom node secret key for use in receiving a [phantom node payment].
```

Comment:
I found this a bit confusing, can we explicitly say "aka node_id or network_key" like we do for `get_node_secret` and describe how this is the secret key which signs invoices?